packages/web3-eth-contract/test/integration/setup.js

```diff
@@ -19,6 +19,6 @@ along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 // eslint-disable-next-line @typescript-eslint/no-require-imports
 require('../config/setup');
 
-const jestTimeout = 15000;
+const jestTimeout = String(process.env.WEB3_SYSTEM_TEST_PROVIDER).includes('ipc') ? 25000 : 15000;
```

Comment:
I would think creating a `JEST_TIMEOUT` (defaulting to `15000`) ENV would be a better approach than this as it's more explicit to the developer that the environment for IPC testing is different

Reply:
it is only for contract tests that's why I use this solution